contracts/aave/AaveV2Contract.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "hardhat/console.sol";
+import "../libraries/Errors.sol";
+import "../interfaces/IPoolV2.sol";
+import "../interfaces/IWETHGateway.sol";
+import "../libraries/PercentageMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+
+/**
+ * @author CRUIZE.
+ * @title Cruize AaveV2Wrapper.
+ * @notice Aave version 2 contract integration to borrow a 20% loan against the user's deposited asset.
+ * 1 - User's deposited asset from the asset pool is deposited into Aave.
+ * 2 - A 20% loan is borrowed in USDC.
+ * 3 - The borrowed USDC is deposited into our USDC pool.
+ */
+contract AaveV2Wrapper is Ownable, ReentrancyGuardUpgradeable {
+    using PercentageMath for uint256;
+
+    //----------------------------//
+    //        State Variable      //
+    //----------------------------//
+    mapping(address => address) public depositAssets;
+
+    uint256 public borrowRatio = 200; // 20% of 1000
+    uint256 private constant USD_DECIMALS = 8;
+    uint256 private constant ETH_DECIMALS = 18;
+
+    IPoolV2 public constant POOL =
+        IPoolV2(0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9);
+
+    IWETHGateway public constant WETHGATEWAY =
+        IWETHGateway(0xcc9a0B7c43DC2a5F023Bb9b738E45B0Ef6B06E04);
+
+    IERC20 public constant BORROW_ASSET =
```

Comment:
why are you loading this here do we really need to load this in contract? are we using this?